swappy.1.scd

```diff
@@ -79,6 +80,7 @@ The following lines can be used as swappy's default:
 - *early_exit* is used to make the application exit after saving the picture or copying it to the clipboard
 - *fill_shape* is used to toggle shape filling (for the rectangle and ellipsis tools) on or off upon startup
 - *auto_save* is used to toggle auto saving of final buffer to *save_dir* upon exit
+- *custom_color* is used to set a default value for the custom color
```

Comment:
@DriesOlbrechts do you think you could add a bit more to the man page? Kind of repeating what format is supported as you did in the description of the MR?